frontend/scss/components/textbook__page.scss

```diff
@@ -48,6 +48,74 @@ main.body {
     @include mq($until: medium) {
       overflow: hidden;
     }
+
+    &__error {
+      display: flex;
+      padding: $spacing-10 $spacing-13;
+  
+      @include mq($until: x-large) {
+        padding: $spacing-10 $spacing-12;
+      }
+
+      @include mq($until: large) {
+        padding: $spacing-09 $spacing-07;
+      }
+
+      @include mq($until: medium) {
+        flex-direction: row;
+        gap: 0;
+        padding-left: $spacing-05;
+        padding-right: 0;
+      }
+
+      @include mq($until: small) {
+        flex-direction: column;
+        padding: $spacing-06 $spacing-05;
+      }
+  
+      &-container {
```

Comment:
Could you take a look at your styles definition? we are using `__` instead of `-`. For example:
```
&-container
```
should be
```
&__container
```